.github/workflows/ci.yml

```diff
@@ -0,0 +1,140 @@
+name: font-ci
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+env:
+  CI: true
+  PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: true
+
+jobs:
+  setup:
+    name: Setup
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.16.1'
+
+    - id: cache-node-modules
+      name: Cache Node.js modules
+      uses: actions/cache@v1
+      with:
+        path: ~/.npm  # npm caches files in ~/.npm
+        key: node-${{ hashFiles('**/package-lock.json') }}
+
+    - name: Install dependencies
+      run: npm install --no-audit
+
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    needs: setup
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.16.1'
+
+    - id: cache-node-modules
+      name: Cache Node.js modules
+      uses: actions/cache@v1
+      with:
+        path: ~/.npm  # npm caches files in ~/.npm
+        key: node-${{ hashFiles('**/package-lock.json') }}
+
+    - name: Install dependencies
+      run: npm install --prefer-offline --no-audit
+
+    - name: Run lint
+      run: npm run lint
+
+  unit:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    needs: setup
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.16.1'
+
+    - id: cache-node-modules
+      name: Cache Node.js modules
+      uses: actions/cache@v1
+      with:
+        path: ~/.npm  # npm caches files in ~/.npm
+        key: node-${{ hashFiles('**/package-lock.json') }}
+
+    - name: Install dependencies
+      run: npm install --prefer-offline --no-audit
+
+    - name: Run tests
+      run: npm run test:
```

Comment:
There's also a syntax error on this line

Reply:
Keep track of the status in your [Actions tab](https://github.com/kushaggarwal/fonts/actions).

Reply:
Okay Thank you